Documentation/Development.md

```diff
@@ -80,6 +80,9 @@ whether you are modifying X10.
 > **Note:** If `swiftc` is not in your `PATH`, you must specify the path to it
   using `-D CMAKE_Swift_COMPILER=`.
 
+> **Note:** If the `ninja` build tool is not in your `PATH`, you must [install
```

Comment:
I think that the phrasing here is odd - if it isn't in the path, you need to add it to your path.  Installation does not guarantee addition to the path (e.g. I may have installed it to `/opt/ninja` which does not get added to your path - at least not by any distribution that I am aware of as `/opt` is not a unified alternate root but rather a dumping ground).